content/chainguard/chainguard-images/how-to-use/use-with-openshift.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "How to Use Chainguard Containers with OpenShift"
+linktitle: "Using With OpenShift"
+type: "article"
+description: "Special considerations for using Chainguard Containers with OpenShift"
+date: 2025-06-17T08:49:31+00:00
+lastmod: 2025-06-17T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "OpenShift", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 008
+toc: true
+---
+
```

Comment:
The "why do we care" kind of statement is still missing at the top. I know you had something to that effect in an earlier draft. But think about who is using this / who cares about this / what does this solution offer. It only needs to be a sentence.